argo-cd-apps/base/all-clusters/external-secrets-operator/external-secrets-operator.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: argoproj.io/v1alpha1
```

Comment:
we are missing an applicationset for cluster-secret-stone in folder argocd-apps/base/all-cluster/cluster-secret-store